### PR TITLE
Text to speech added successfully.

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#271 
Closes: # (issue number)
#271 
## Proposed changes

Brief description of what is fixed or changed
Using gTTs library I have implemented text to speech. The user can enter the text an audio file will be saved in wplay/audioFiles. The user can use this feature in chatting, message-schedule, and message blast. The user has to type command #_TTS then the user has to enter the text it wants to convert to audio file then a list of languages supported will be printed it can enter the code. There also many more language option available but I have added some common and all Indian languages. Then the audio file will be saved and automatically the file attachment option will open the user can select the file and it will be sent.
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot from 2020-05-24 23-57-43

Screenshot from 2020-05-24 23-57-30

WhatsApp Image 2020-05-25 at 12 05 29 AM

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- wplay/utils/io.py  1
- wplay/terminal_chat.py  1
- wplay/text_to_speech.py  2
         

Clones added
============
- wplay/save_chat.py  3
- wplay/terminal_chat.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5576450&bid=18281861)

File: wplay/save_chat.py [hunk 1]
```diff
@@ -246,4 +246,4 @@ def runMain(mode, asset, bID):
             if len(drives) > 1:
                 print('Backup: ' + str(i))
                 folder = 'WhatsApp-' + str(i)
-            getMultipleFiles(drive[1], folder)
\ No newline at end of file
```
Comment (rpotter12) [inline wplay/save_chat.py:249]: Codacy Issue found: [String statement has no effect](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5576450)

File: wplay/text_to_speech.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        list_laguages=['bn: Bengali', 'de: German', 'en: English','es: Spanish','fr: French','gu: Gujarati','hi: Hindi',
+        'it: Italian','ja: Japanese','kn: Kannada','ko: Korean','ml: Malayalam','mr: Marathi','pt-br: Portuguese (Brazil)','ru: Russian',
+        'ta: Tamil','te: Telugu','ur: Urdu',]
+
+        print('Choose a code for language of your choice from the following list\n')
+        print(list_laguages)
+
+        # Language in which you want to convert
+        language = input("\n\nEnter the language you want to convert to audio file: ")
+
```
Comment (rpotter12) [inline wplay/text_to_speech.py:31]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5576450)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/text_to_speech.py  2
- wplay/terminal_chat.py  1
- wplay/utils/io.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5576450&bid=18281861)

**Comment by SinghHimadri** [comment]
@rpotter12 please review my PR.

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/text_to_speech.py  2
- wplay/terminal_chat.py  1
- wplay/utils/io.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5576450&bid=18281861)

**Comment by SinghHimadri** [comment]
@rpotter12 I have resolvded the conflicts please review my PR.

**Pull request merged**